src/components/dashboard/content/list/DashboardFaultTreeList.tsx

```diff
@@ -52,7 +52,7 @@ const DashboardFaultTreeList = () => {
     return (
         <React.Fragment>
             <ImageList className={classes.gridList} cols={6}>
-                {faultTrees.filter(f => !!f && !!f.iri).map((tree) => {
+                {faultTrees.map((tree) => {
```

Comment:
Make sure that it is ok to remove this filter. You removed it elsewhere in this file as well.

Reply:
@blcham 
Yes I believe it is ok to remove the filter. It was added by me in #132. 
In this PR #152, I fix processing of json-ld in src/util/JsonLdUtils.tsx according to TermIt so that the filter is not needed anymore.